float_time_seconds/static/src/js/float_time_seconds.js

```diff
@@ -1,4 +1,4 @@
-openerp.float_time_seconds = function(instance){
+odoo.float_time_seconds = function(instance){
 
     original_format_value = instance.web.format_value;
     instance.web.format_value = function (value, descriptor, value_if_empty) {
```

Comment:
The [original format_value function](https://github.com/OCA/OCB/blob/10.0/addons/web/static/src/js/framework/formats.js#L20), that we are overriding here, has moved from "instance.web.format_value" to "instance.web.formats.format_value".

If we also follow the new best practice ([example](https://github.com/OCA/web/blob/10.0/web_menu_navbar_needaction/static/src/js/web_menu_navbar_needaction.js#L21-L36)) to extend Javascript things, we would write something like:

```
openerp.float_time_seconds = function(instance)
{
    instance.web.formats.include({
        format_value: function(value, descriptor, value_if_empty) {
            switch......
            return this._super.apply(this, arguments);  // call original function
        },
        parse_value: function(value, descriptor, value_if_empty) {
            switch......
            return this._super.apply(this, arguments);  // call original function
        },
    }
}
```